crates/forge/tests/e2e/workspaces.rs

```diff
@@ -0,0 +1,306 @@
+use std::path::PathBuf;
+
+use assert_fs::fixture::PathCopy;
+use indoc::indoc;
+
+use crate::e2e::common::runner::runner;
+
+#[test]
+fn run_without_arguments() {
+    let temp = assert_fs::TempDir::new().unwrap();
+    temp.copy_from("tests/data/hello_workspaces", &["**/*.cairo", "**/*.toml"])
+        .unwrap();
+
+    let snapbox = runner();
+
+    snapbox
+        .current_dir(&temp)
+        .assert()
+        .success()
+        .stdout_matches(indoc! {r#"
+        [..]Compiling[..]
+        [..]Finished[..]
+        Collected 3 test(s) and 2 test file(s)
+        Running 1 test(s) from hello_workspaces package
```

Comment:
I think these outputs are a little bit confusing if we say that only some of tests are run from the xyz package, if in fact all of them are (just that some are not in the src, but in tests for example).

Reply:
You mean `from XYZ package` part? We did this change way before this PR, it is basically an alternative for `from src/lib.cairo` which is even more missleading, since we look for tests in `src` folder only using the `lib.cairo` entrypoint (and then tests are found using the module tree)

Reply:
Yes, the `from XYZ package` part. The problem I have with this is this just suggest that tests outside `src` are not really part of the package which is kind of true but also it isn't.

Reply:
Still, I guess it's outside of the scope of this pr

Reply:
What about moving package name to the previous line, and adding some kind of separator like:
```
Collected N test(s) and N test file(s) from XYZ package
Running N' inline test(s)
...
Running N" test(s) from tests/test.cairo
Tests: N passed, 0 failed, 0 skipped

Collected M test(s) and M test file(s) from ABC package
Running M' inline test(s)
...
Running M" test(s) from tests/nested/test.cairo
Tests: M passed, 0 failed, 0 skipped
```

Reply:
I think this could clear up the confusion. What do you thing @piotmag769?

Reply:
Fine for me